src/vcpkg/base/files.cpp

```diff
@@ -1119,7 +1119,7 @@ namespace vcpkg
     // This implementation does collapse slashes because we primarily use it for shiny display purposes.
     void Path::make_preferred()
     {
-        char* first = &m_str[0];
+        char* first = m_str.data();
```

Comment:
It was probably done this way before because non-const `basic_string::data` is a C++17 feature. (No change requested)